hal/linux/spektrum_satellite.c

```diff
@@ -12,7 +12,7 @@
 // #include "joystick.h"
 
 #include <stdlib.h>
-#include <stdio.h>
+// #include <stdio.h>

```

Comment:
if not needed we could remove.

Besides, isn't there cstdlib for the upper one ?